ngraph/core/builder/src/builder/autobroadcast.cpp

```diff
@@ -298,6 +298,11 @@ Output<Node> legacy_broadcast_for_binary_operation(const Output<Node>& left,
         return right;
     }
 
+    // WA to fix RNNSequence tests
```

Comment:
Could we extend this comment?

Reply:
The author does not remember the meaning of this change therefore it was reverted.